lib/ui/text.dart

```diff
@@ -791,7 +796,10 @@ ByteData _encodeStrut(
 
   return ByteData.view(data.buffer, 0,  byteCount);
 }
-
+/// See also:
```

Comment:
nit: newline before this

Is this class missing public documentation?  /cc @GaryQian